dcos_test_utils/dcos_cli.py

```diff
@@ -18,7 +18,7 @@
 
 log = logging.getLogger(__name__)
 
-DCOS_CLI_URL = os.getenv('DCOS_CLI_URL', 'https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.11/dcos')
+DCOS_CLI_URL = os.getenv('DCOS_CLI_URL', 'https://downloads.dcos.io/cli/testing/binaries/dcos/linux/x86-64/master/dcos')
```

Comment:
I am not reviewing this PR and I know it is still `WIP` but I'll just post here ask to not use `master` version, which is a moving target and rather pin this URL to a specific version.